tripper/namespace.py

```diff
@@ -0,0 +1,181 @@
+"""Provides a simple representation of namespaces."""
```

Comment:
Are there any changes in this content after transferring it here from `triplestore.py`? It might be nice to just to a PR (or a dedicated commit) that has _only_ the split up and move within it, and then a subsequent PR (or dedicated commit) to implement any changes.

Reply:
Maybe some bug fixes

Reply:
Which?